charts/victoria-metrics-cluster/templates/vmstorage-pdb.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.vmstorage.enabled -}}
```

Comment:
Hi there is no `{{ end }}` statement

Reply:
Refactored it to look more like > https://github.com/VictoriaMetrics/helm-charts/blob/d1e0060d64cc17789a9c08b6dde98314c9cd0c42/charts/victoria-metrics-cluster/templates/vmstorage-service-monitor.yaml#L1